Sources/WalletConnectSign/Engine/Common/SessionEngine.swift

```diff
@@ -71,14 +74,18 @@ final class SessionEngine {
         }
         let chainRequest = SessionType.RequestParams.Request(method: request.method, params: request.params)
         let sessionRequestParams = SessionType.RequestParams(request: chainRequest, chainId: request.chainId)
-        try await networkingInteractor.request(.wcSessionRequest(sessionRequestParams), onTopic: request.topic)
+
+        let payload = WCRequest.sessionRequest(sessionRequestParams)
+        let rpcRequest = RPCRequest(method: WCRequest.Method.sessionRequest.method, params: payload)
+        try await networkingInteractor.request(rpcRequest, topic: request.topic, tag: WCRequest.Method.sessionRequest.requestTag)
     }
 
     func respondSessionRequest(topic: String, response: JsonRpcResult) async throws {
         guard sessionStore.hasSession(forTopic: topic) else {
             throw Errors.sessionNotFound(topic: topic)
         }
-        try await networkingInteractor.respond(topic: topic, response: response, tag: 1109) // FIXME: Hardcoded tag
+        // TODO: ???
+//        try await networkingInteractor.respond(topic: topic, response: response, tag: 1109) // FIXME: Hardcoded tag
```

Comment:
does tests pass now?
we can't comment it out, that's the essence of the protocol